mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -414,7 +411,11 @@ properties:
     type: NestedObject
     description: |
       Represents the workload policy.
-    min_version: 'beta'
+    conflicts:
+      - instance_schedule_policy
+      - snapshot_schedule_policy
+      - disk_consistency_group_policy
+      - group_placement_policy
```

Comment:
Can you remove the `conflicts`? Adding the validation to a minor provider release is a breaking change according to https://googlecloudplatform.github.io/magic-modules/breaking-changes/breaking-changes/#making-validation-more-strict. 

You can add it to the next provider major release provider 7.0.0.

Reply:
Will the API return error if `workloadPolicy` and any other fields in this list are provided in the API request?